x-pack/plugins/infra/public/hooks/use_http_request.tsx

```diff
@@ -20,7 +20,8 @@ export function useHTTPRequest<Response>(
   decode: (response: any) => Response = (response) => response,
   fetch?: HttpHandler,
   toastDanger?: (input: ToastInput) => void,
-  abortable = false
+  abortable = false,
+  fetchOptions?: Omit<HttpFetchOptions, 'body' | 'method' | 'signal'>
```

Comment:
There was no way to pass URL query for a GET request, so I needed to add this additional option here.